src/libs/fetchPronouns.js

```diff
@@ -86,6 +89,8 @@ async function fetchStatus(statusID) {
 
 	let status = await response.json();
 
+	if (!response.ok) return null;
```

Comment:
```suggestion
	if (!response.ok) return null;

	let status = await response.json();
```

Switched those lines, because otherwise the `response.json()` promise might crash because the response is not okay.